Assets/Scripts/Interactable/NPCInteractable.cs

```diff
@@ -1,10 +1,13 @@
+using System.Collections.Generic;
 using UnityEngine;
 
 namespace Interactable
 {
     public class NPCInteractable : MonoBehaviour, IInteractable
     {
-        [SerializeField] private string interactText;
+        [SerializeField] protected string npcID;
+        [SerializeField] protected List<string> interactText;
+        public int dialogIndex { get; set; }
```

Comment:
use `private` instead of `protected`

Reply:
`dialogIndex` use `private` instead of `public`

Reply:
fix